src/components/admin/ActionButtons.vue

```diff
@@ -1,19 +1,79 @@
 <template>
-  <q-td :props="props" class="text-right">
-    <q-btn color="green" class="q-mx-sm">Approve</q-btn>
+  <div class="flex justify-center">
     <q-btn
+      round
+      flat
+      color="green"
+      class="q-mx-sm"
+      icon="check"
+      @click="approveApplication"
```

Comment:
You may want to use this as to `viewApplication` which leads to `approveApplication`